doc/source/conf.py

```diff
@@ -90,6 +90,10 @@
 # The current released version
 rst_prolog = """
 .. |current_flwr_version| replace:: 1.10.0
+.. |latest_version_docker_sha| replace:: 985c24b2b337ab7f15a554fde9d860cede95079bcaa244fda8f12c0805e34c7d
```

Comment:
```suggestion
.. |latest_flwr_superlink_docker_digest| replace:: xxxc51d889c
```
The digest for `flwr/superlink:1.10.0` is different for me (ends with `c51d889c`). Should we update this? Also, for clarity, I think we should use a more explicit tag like `|latest_flwr_superlink_docker_digest|`?

Reply:
Good catch! I forgot to update it.